.github/workflows/entrypoint.yml

```diff
@@ -0,0 +1,29 @@
+name: Build doc
+
+on:
+  push:
+    paths-ignore:
+      - ".github/**"
+      - "**.md"
+  pull_request:
+    paths-ignore:
+      - ".github/**"
+      - "**.md"
+
+
+jobs:
+  build:
+    name: Build doc
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: true
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install requirements
+        run: python -m pip install --user -r ./requirements.txt
+      - name: Build docs
+        run: make html
```

Comment:
No need to install make in the runner contexte ?

Reply:
Indeed, didn't think about it, but it works oddly enough 🤷